app/javascript/components/Pages/ListEmotions.js

```diff
@@ -47,10 +47,9 @@ function ListEmotions({ data,  setData , saveDataToDb, steps, service}) {
   }
 
   const rangeFormat = (tp) => {
-    let start_date = new Date(tp.start_date)
-    let end_date = new Date(tp.end_date)
-    let month = end_date.toLocaleString('default', {month: 'long'}).slice(0,3)
-    return `${start_date.getDate()}`.padStart(2, '0') + '-' + `${end_date.getDate()}`.padStart(2, '0') + ' ' + month
+    let dueDate = new Date(tp.due_date)
+    let month = dueDate.toLocaleString('default', {month: 'long'}).slice(0,3)
```

Comment:
those variables are **const** ones. You do not change those values after. Use **const** on variables declaration.

Reply:
I fixed. Thanks)